app/src/main/java/com/discordtime/gpts/repository/PlaceRepository.kt

```diff
@@ -2,9 +2,12 @@ package com.discordtime.gpts.repository
 
 import androidx.lifecycle.LiveData
 import com.discordtime.gpts.listplaces.model.Place
+import com.discordtime.gpts.remote.FirestoreAPI
 import com.discordtime.gpts.remote.RemoteAPI
 
-class PlaceRepository(val remoteAPI: RemoteAPI<Place>): IPlaceRepository {
+class PlaceRepository: IPlaceRepository {
+
+    private val remoteAPI: RemoteAPI<Place> = FirestoreAPI()
 
     override fun getPlaces(): LiveData<List<Place>> {
         return remoteAPI.get()
```

Comment:
As I mentioned above, I think we could have a mapper here that would take care of the transformation from what is returned from the remote api. I think we could put it the mapper on the constructor and have it to be injected on the Koin module.